test/e2e_node/services/kubelet_test.go

```diff
@@ -0,0 +1,142 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package services
```

Comment:
See this https://github.com/kubernetes/kubernetes/blob/master/hack/make-rules/test.sh#L59-L71.

You may want to add:
```
        -o -path './test/e2e_node/services/*' \
```

Reply:
Done